docs/BreakingChanges/168 DataContractJsonSerializer incompatible with ES6 (and V8).md

```diff
@@ -0,0 +1,33 @@
+## 168: DataContractJsonSerializer incompatible with ES6 (and V8)
+
+### Scope
+Edge
+
+### Version Introduced
+4.7
+
+### Source Analyzer Status
+Planned
+
+### Change Description
+.NET 4.7 fixed the compatibility issue between the escaped strings for some special control characters generated by DataContractJsonSerializer and ECMAScript V6. 
+
+- [x] Quirked
+- [ ] Build-time break
+
+### Recommended Action
+Prior to 4.7, this change is disabled by default. You can enable this fix by adding the following line to the `<runtime>` section of the app.config or web.config file:
```

Comment:
Change wording: "Prior to 4.7, this change ..." to "Prior to .NET 4.7, this change ... "